assets/src/stories-editor/blocks/amp-story-cta/deprecated.js

```diff
@@ -73,6 +91,50 @@ CallToActionSaveV121.propTypes = {
 	} ).isRequired,
 };
 
+/**
+ * Deprecated save function which was used last for plugin version 1.3.0
+ *
+ * @param {Object} attributes Attributes.
+ * @return {*} CTA save.
+ */
+const CallToActionSaveV130 = ( { attributes } ) => {
+	const {
+		anchor,
+		btnPositionLeft,
+		btnPositionTop,
+		text,
+		url,
+	} = attributes;
+
+	const className = getClassNameFromBlockAttributes( { ...attributes, className: 'amp-block-story-cta__link' } );
+	const styles = getStylesFromBlockAttributes( attributes );
+
+	styles.top = btnPositionTop ? `${ btnPositionTop }%` : undefined;
+	styles.left = btnPositionLeft ? `${ btnPositionLeft }%` : undefined;
+
+	return (
+		<amp-story-cta-layer id={ anchor ? anchor : getUniqueId() }>
```

Comment:
I don't love conditional logic here. Can we just make a const the line above?

Reply:
Note that this is a deprecation which is an exact copy of the `save` function from the plugin version `1.3.0`.

Let's keep it as it is and not make changes "to the past".

Reply:
Fair enough